forge-gui/res/cardsfolder/upcoming/urgent_necropsy.txt

```diff
@@ -0,0 +1,10 @@
+Name:Urgent Necropsy
+ManaCost:2 B G
+Types:Instant
+A:SP$ Destroy | Cost$ CollectEvidence<X> 2 B G  | ValidTgts$ Artifact | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Select up to one target artifact | SubAbility$ DBDestroyCreature | SpellDescription$ Destroy up to one target artifact, up to one target creature, up to one target enchantment, and up to one target planeswalker.
+SVar:DBDestroyCreature:DB$ Destroy | ValidTgts$ Creature | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Select up to one target Creature | SubAbility$ DBDestroyEnchantment
+SVar:DBDestroyEnchantment:DB$ Destroy | ValidTgts$ Enchantment | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Select up to one target Enchantment | SubAbility$ DBDestroyPlaneswalker
+SVar:DBDestroyPlaneswalker:DB$ Destroy | ValidTgts$ Planeswalker | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Select up to one target Planeswalker
```

Comment:
we just had the same scenario...
one keyword action = it needs to happen simultaneous